### PR TITLE
Add the Pokemon faker #46

### PR DESCRIPTION
**Pull request by tehmantra**
See issue #46

### CODE REVIEW

File: src/FakerDotNet/Fakers/PokemonFaker.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        private static readonly PokemonData Data = new PokemonData();
+        
+        private readonly IFakerContainer _fakerContainer;
+
+        public PokemonFaker(IFakerContainer fakerContainer)
+        {
+            _fakerContainer = fakerContainer;
+        }
+
+
```
Comment (mrstebo) [inline src/FakerDotNet/Fakers/PokemonFaker.cs]: Can you remove the extra carriage return? The rest of the classes don't have this format

File: doc/name.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Faker.Name
 
 ```cs
-Faker.Name.Name()             //=> "Tyshawn Johns Sr."
+Faker.Name.Name()           //=> "Tyshawn Johns Sr."
```
Comment (mrstebo) [inline doc/name.md]: Could you revert the changes on this file? It doesn't relate to the Pokemon feature.

File: src/FakerDotNet/Fakers/PokemonFaker.cs (hunk 1 above)
Comment (tehmantra) [reply src/FakerDotNet/Fakers/PokemonFaker.cs]: Done

**Comment by mrstebo** [comment]
Excellent! Thanks for the contribution @tehmantra :tada:

**Comment by mrstebo** [comment]
Resolved #46
